scholly/Dockerfile

```diff
@@ -1,19 +1,19 @@
-FROM node:alpine
+FROM node
 
+USER node
+
+RUN mkdir -p '/home/node/app'
```

Comment:
Why do we need to make a directory here? We don't seem to do it in any of the other Dockerfiles (for `client` and `api`)

Reply:
Great question, so:
1. Containers by default execute as `root` user
2. The `WORKDIR` command creates non-existing directories and sets the owner to `root`
3. The `USER` directive specifies that the container should run as `node`
4. Scholly creates that `build` folder inside this directory when building an application
5. `node` user does not have more permissions than `root` so we have a permission conflict
6. This change makes it so that `node` owns this directory

Why do we want to execute as `node` user and not `root`? It's about security: see point 2 in https://snyk.io/blog/10-docker-image-security-best-practices/

https://docs.docker.com/engine/reference/builder/#user

Ref: https://github.com/nodejs/docker-node/issues/740

Reply:
Right great explanation. So if I understand correctly we could also alternatively use something like `chmod` or `chown` to give `node` permission to `/home/node/app` right?

Reply:
Yep, that’d work too.